### PR TITLE
Fix script for new nozzle driver location

### LINKED ISSUE #11: Timings failing due to merge conflict
When merging `mirgecom@y1_production` into `mirgecom@main` the conflict below kills the timing script.  [This _MIRGE-Com_ PR](https://github.com/illinois-ceesd/mirgecom/pull/307) fixes.

```
++ git rev-parse y1_production
+ git checkout main
Already on 'main'
Your branch is up to date with 'origin/main'.
+ git switch -c temp
Switched to a new branch 'temp'
+ git merge y1_production --no-edit
Auto-merging mirgecom/simutil.py
Auto-merging mirgecom/initializers.py
Auto-merging mirgecom/boundary.py
CONFLICT (content): Merge conflict in mirgecom/boundary.py
Automatic merge failed; fix conflicts and then commit the result.
```

### PR DESCRIPTION
**Pull request by MTCam**
Fixes #11

### CODE REVIEW

File: time-y1-nozzle.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ git merge y1_production --no-edit
 # --- Grab the nozzle driver repo
 rm -Rf CEESD-Y1_nozzle
 git clone https://github.com/anderson2981/CEESD-Y1_nozzle.git
-cd CEESD-Y1_nozzle/startup
+cd CEESD-Y1_nozzle/timing_run
```
Comment (inducer) [inline time-y1-nozzle.sh:50]: Whose job is it to maintain the `timing_run` script? (I.e. who makes sure it doesn't fall behind the actual driver?)

Comment (MTCam) [reply time-y1-nozzle.sh:50]: I vote for the owner of CEESD-Y1_nozzle, (@anderson2981).  

From my perspective he should be able to drop anything he wants to add to timing in that directory with a common form for the setup parameters and get it timed.  But that's another PR or so down the road (if you like it).

Comment (inducer) [reply time-y1-nozzle.sh:50]: Why not just time (a slightly patched version of) the actual script in the driver?

Comment (MTCam) [reply time-y1-nozzle.sh:50]: I'm a little bit confused.  Isn't that what we do?

Comment (anderson2981) [reply time-y1-nozzle.sh:52]: At least in my current workflow there is no "one" driver that can be reliably timed. I make effort to keep the version in timing_run up to date with the current state of the art, which should be reflected in baseline. I did some restructuring of this particular driver repo, as it was becoming too cluttered to see what was going on.

The proposed change is no different that the previous approach, we were timing a crusty old version of the driver that happened to live in a folder named startup, which was my first stab at getting the nozzle to run.

**Pull request merged**